contract.md

```diff
@@ -4,6 +4,10 @@
 
 _At a high level, what do you most want to get out of this project, and why?_
 
+In my time at Mudd, I've spent a lot of time learning hard CS skills, such as algorithms and data structures. However, more subjective qualities of a CS project can be just as impactful on the quality of the final product. By comparison, the Mudd curriculum has done a lot less with this side of the field. With this in mind, I really want to work on polishing the user experience through effective language design. To do this properly, I'll need a good sense for how people interact with my language in each iteration. As a result, in-class and outside interviews and user tests will be very useful for me. 
+
+I also want to gain some experience taking the time to reflect and revise my work once it is functional. I feel that in computer science we're always rushing to some end goal without taking stock of what was accomplished and how it could be improved. And since this class focuses so heavily on language design, taking the time to consider and improve the final product is especially worthwhile. That is why I've scheduled my intermediate goals to have a complete implementation by week 3 (more on that in *Time management*) so I can do additional testing and revisions with the remainder of my time.
```

Comment:
Including a full week testing window seems like a really good way to practice your skills at revising a program to improve its user experience